config/pr.yml

```diff
@@ -9,14 +9,28 @@ base_redirect: http://proconsortium.org
 products:
 - pr.owl: ftp://ftp.proconsortium.org/databases/ontology/pro_obo/pro_reasoned.owl
 - pr.obo: ftp://ftp.proconsortium.org/databases/ontology/pro_obo/pro_reasoned.obo
-- pr-asserted.owl: ftp://ftp.proconsortium.org/databases/ontology/pro_obo/pro.owl
-- pr-asserted.obo: ftp://ftp.proconsortium.org/databases/ontology/pro_obo/pro.obo
+- pr-asserted.owl: ftp://ftp.proconsortium.org/databases/ontology/pro_obo/pro_nonreasoned.owl
+- pr-asserted.obo: ftp://ftp.proconsortium.org/databases/ontology/pro_obo/pro_nonreasoned.obo
 
 term_browser: custom
 
 entries:
+- prefix: /release/
+  replacement: ftp://ftp.proconsortium.org/databases/ontology/pro_obo/old_files/release_
+  tests:
+  - from: /release/52.0/pr.obo
```

Comment:
Unfortunately this won't work. You'll get `ftp://ftp.proconsortium.org/databases/ontology/pro_obo/old_files/release_52.0/pr.obo` without `_reasoned`.

Reply:
I was afraid of that. Would it be possible to declare--prior to the above--something like
- exact: pr.owl
   replacement: pro_reasoned.owl
(here hoping that the lack of slash will allow just the last part to get replaced)